examples/06_wifi_scan/main/app_main.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * MaiKe Labs (2016 - 2026)
```

Comment:
I don't understand 2026...? I'm 100% OK with you crediting yourself and MaiKe Labs here, but could you please combine these into one line, ie
```
WiFi scan example

Written by Jack Tan of MaiKe Labs <jiankemeng@gmail.com>

This example code is in the Public Domain ...
```